Formula/lftp.rb

```diff
@@ -0,0 +1,34 @@
+class Lftp < Formula
+  desc "Sophisticated file transfer program"
+  homepage "https://lftp.tech"
+  url "https://lftp.yar.ru/ftp/lftp-4.7.5.tar.bz2"
+  mirror "ftp://ftp.st.ryukoku.ac.jp/pub/network/ftp/lftp/lftp-4.7.5.tar.bz2"
+  sha256 "90f3cbc827534c3b3a391a2dd8b39cc981ac4991fa24b6f90e2008ccc0a5207d"
+
+  depends_on "readline"
+  depends_on "openssl"
+  depends_on "libidn"
+
+  # On Yosemite, the system's openssl gets chosen over ours, and it is too old
```

Comment:
Thanks for reporting, and adding a note to the formula. I don't think it'll be a blocker here, but it'll be nice to remove that line eventually :smile: